examples/http-retry/http_retry.server.out

```diff
@@ -2,8 +2,8 @@
 #`http_retry.bal` file and run the `ballerina run` command to start the server.
 $ ballerina run http_retry.bal.bal
 Initiating service(s) in 'http_retry.bal'
-ballerina: started HTTP/WS endpoint 0.0.0.0:8080
-ballerina: started HTTP/WS endpoint 0.0.0.0:9090
+[ballerina/http] started HTTP/WS endpoint 0.0.0.0:8080
+[ballerina/http] started HTTP/WS endpoint 0.0.0.0:9090
 2018-04-23 15:31:07,872 INFO  [] - Request received from the client to delayed service.
 2018-04-23 15:31:12,888 INFO  [] - Request received from the client to delayed service.
 2018-04-23 15:31:12,905 ERROR [] - Error sending response from mock service : {message:"Remote client closed the connection before completing outbound response", cause:null, statusCode:0}
```

Comment:
Can you double check what's logged for the error here? Ideally, `cause` shouldn't come up here since it's an optional field and doesn't seem to be explicitly set.

Reply:
I think cause comes as null because we are explicitly passing null from createError(...) method[1].

[1]. https://github.com/ballerina-platform/ballerina-lang/blob/master/bvm/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/BLangVMErrors.java#L86